spec/javascripts/helpers/jasmine-jquery.js

```diff
@@ -0,0 +1,841 @@
+/*!
```

Comment:
I did some googling but but quite sure how to include that in a Rails project using CDN, since that might require adding a script tag with that CDN to `application.html.erb` and I don't think it's good to include test dependency into application code?